lib/sync/storage-manager-listener.js

```diff
@@ -0,0 +1,36 @@
+let cachedStore;
+let cachedListener;
+
+module.exports = function getStorageManagerListener(store) {
+  if (cachedListener && (!store || store === cachedStore)) { return cachedListener; }
+
+  if (!store || (typeof store.dispatch !== 'function')) { throw new Error('You must pass in a redux store the first time you call getStorageManagerListener'); }
+
+  cachedStore = store;
```

Comment:
Perhaps we add this only for specs? This could be problematic in source if we somehow end up with multiple stores that get passed in.